control_panel_api/tests/test_permissions.py

```diff
@@ -386,3 +386,53 @@ def test_update_as_normal_user_responds_403(self):
         response = self.client.put(
             reverse('s3bucket-detail', (self.s3bucket_1.id,)), data)
         self.assertEqual(HTTP_403_FORBIDDEN, response.status_code)
+
+
+class K8sPermissionsTest(APITestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.superuser = mommy.make(
+            'control_panel_api.User',
+            auth0_id='github|user_1',
+            is_superuser=True,
+        )
+        self.normal_user = mommy.make(
+            'control_panel_api.User',
+            auth0_id='github|user_2',
+            is_superuser=False,
+        )
+
+    def test_when_not_authenticated_responds_403(self):
+        response = self.client.get('/k8s/something')
+        self.assertEqual(HTTP_403_FORBIDDEN, response.status_code)
+
+    @patch('kubernetes.client.configuration')
+    @patch('kubernetes.config.load_incluster_config')
+    @patch('requests.request')
+    def test_superuser_can_do_anything(self, mock_request, mock_load_config, mock_config):
+        self.client.force_login(self.superuser)
+
+        mock_request.return_value.status_code = 200
+
+        response = self.client.get('/k8s/something')
+        self.assertEqual(HTTP_200_OK, response.status_code)
+
+    @patch('requests.request')
```

Comment:
If this is testing a 403 does it need the patch, does it even reach the requests execution?